### ISSUE TITLE
ohem loss

### ISSUE THREAD

**Issue by gbyy422990** [yassouali/CCT#35]
hi, Its really a nice work.

CCT does NOT work when I changed supervised loss function from cross-entropy to OHEM, the mean-IoU on eval dataset seems like really close between full-sup and semi-sup, I dont know why this happen? And by the way, have u ever tried ohem loss?

**Comment by yassouali** [comment]
Hi @gbyy422990 
unfortunately, I did not test the method with ohem loss, so I can't say for sure why that is the case. But maybe you'll need to modify the hyperparameters a little bit to find better results. Or maybe implement a CCT loss with a hard example mining instead of computing it over the whole unlabeled batch.

Good luck.
